src/test/java/org/broadinstitute/hellbender/tools/walkers/MethylationTypeCallerIntegrationTest.java

```diff
@@ -0,0 +1,60 @@
+package org.broadinstitute.hellbender.tools.walkers;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.testutils.IntegrationTestSpec;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.File;
+
+public class MethylationTypeCallerIntegrationTest extends CommandLineProgramTest {
+
+    // If true, update the expected outputs in tests that assert an exact match vs. prior output,
+    // instead of actually running the tests. Can be used with "./gradlew test -Dtest.single=MethylationTypeCallerUnitTest"
+    // to update all of the exact-match tests at once. After you do this, you should look at the
+    // diffs in the new expected outputs in git to confirm that they are consistent with expectations.
+    public static final boolean UPDATE_EXACT_MATCH_EXPECTED_OUTPUTS = false;
+
+    public static final String TEST_FILES_DIR = toolsTestDir + "walkers/MethylationTypeCaller/";
+    public static final String TEST_FILES_INPUT_REFERENCE_DIR = largeFileTestDir + "GRCm38_primary_assembly_genome/";
+
+    /*
+     * Test that in VCF mode we're consistent with past  results
+     */
+    @Test(dataProvider="getMethylationTypeCallerTestInput")
+    public void testVCFIsConsistentWithPastResults(final String inputFileName, final String referenceFileName) throws Exception {
+        Utils.resetRandomGenerator();
+
+        final File outputVCF = createTempFile("testVCFIsConsistentWithPastResults", ".vcf");
+        final File expectedVCF = new File(TEST_FILES_DIR, "chr14.unique_reads.methylC_seq.vcf");
```

Comment:
Not sure what "unique_reads" or the "C" means in the context of these file names (?), but it might be helpful to add "expected" to the (on disk) file name of the output vcf, i.e., `expected.chr14.unique_reads.methylC_seq.vcf`.

Also, you can just call `getTestFile("expected.chr14.unique_reads.methylC_seq.vcf")` here (see my other comment on this).

Reply:
the C in methylC_seq refers to a particular assay/pipeline that generates this BAM/VCF